CMakeLists.txt

```diff
@@ -93,6 +93,7 @@ endif()
 option(LLAMA_ACCELERATE                      "llama: enable Accelerate framework"               ON)
 option(LLAMA_BLAS                            "llama: use BLAS"                                  OFF)
 option(LLAMA_LLAMAFILE                       "llama: use llamafile SGEMM"                       ${LLAMA_LLAMAFILE_DEFAULT})
+option(LLAMA_AARCH64                         "llama: use aarch64 optimized GEMM & GEMV kernels" ON)
```

Comment:
No need for this option - add the `ggml-aarch64` sources directly to the `ggml` target

Reply:
We removed the `LLAMA_AARCH64` option in the latest commit and added the `ggml-aarch64` sources directly to the `ggml` target as suggested.